### ISSUE TITLE
Poor documentation.

### ISSUE THREAD

**Issue by grimz11** [sindresorhus/electron-serve#25]
You should consider that someone just want a detailed explaination on how this library works. I just learned Electron and I don't know how to implement this. Can you at least give more detail to this one?

``` typescript
import { app, BrowserWindow} from "electron";
import * as path from "path";
// import * as url from "url";
const serve = require('electron-serve');

let mainWindow: Electron.BrowserWindow | null;

const loadURLs = serve({directory: path.join('file:///', __dirname, "../build/index.html")});

  async function createWindow() {
  await app.whenReady();
  // Create the browser window.electron
  mainWindow = new BrowserWindow({
    height: 600,
    webPreferences: {
      webSecurity: false,
      preload: path.join(__dirname, "preload.js"),
    },
    width: 1400,
  });
  console.log(path.join('://-', __dirname, "../build/index.html"));
  console.log('LoadUrl', {loadURLs});
  
  await loadURLs(mainWindow.loadURL(path.join('://', __dirname, "../build/index.html")));  

  mainWindow.webContents.on('did-start-navigation', () => {
    //if()
    // mainWindow.loadFile(path.join(__dirname, "index.html#/user/login"));
    // mainWindow.loadFile(path.join(__dirname, "index.html#/"));
  });

  // Open the DevTools.
  mainWindow.webContents.openDevTools();

  // Emitted when the window is closed.
  mainWindow.on("closed", () => {
    // Dereference the window object, usually you would store windows
    // in an array if your app supports multi windows, this is the time
    // when you should delete the corresponding element.
    mainWindow = null;
  });
  
}```

Look what I'm doing here. :)

**Comment by sindresorhus** [comment]
Can you please elaborate on what kind of details you think are missing?

**Comment by sindresorhus** [comment]
I don't intend to document how Electron itself works. It's expected that you familiarize yourself with Electron through the official docs.

**Comment by grimz11** [comment]
Do we need electron-store dependency 
or other libarary to make this work?

**Comment by sindresorhus** [comment]
I don't understand the question. Make *what* work?

**Comment by grimz11** [comment]
Okay, as what I have seen from the repo that uses your library. They have some kind of boilerplate that uses "electron-store here's the <a href="https://github.com/LLGames/DesktopLauncher">link</a>. Do I need to use it as well? My problem is that I embedded my react app to electron, it doesn't work when i build it. The homepage work in first load or when I reload it but when I navigate away  from the homepage I got ERR_File_notfound in my network tab E://. And I've come to have read that this is an issue to SPA and I found your library. How do I implement this?
